src/wallet/rpcwallet.cpp

```diff
@@ -3214,9 +3214,8 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             RPCHelpMan{"bumpfee",
                 "\nBumps the fee of an opt-in-RBF transaction T, replacing it with a new transaction B.\n"
                 "An opt-in RBF transaction with the given txid must be in the wallet.\n"
-                "The command will pay the additional fee by decreasing (or perhaps removing) its change output.\n"
-                "If the change output is not big enough to cover the increased fee, the command will currently fail\n"
-                "instead of adding new inputs to compensate. (A future implementation could improve this.)\n"
+                "The command will pay the additional fee by reducing change outputs or adding inputs when necessary. It may add a new change output if one does not already exist.\n"
+                "If `totalFee` is given adding inputs is not supported, so there must be a single change output that is big enough or it will fail.\n"
```

Comment:
In commit "generalize bumpfee to add inputs when needed" (d5c0e53264a7eaf72c2691e3c19ce9fc60d3acf6)

I think this needs a comma after "given" to be grammatical (sorry to be pedantic, but I did double-check https://www.ego4u.com/en/cram-up/writing/comma?10  :bowtie:).

Reply:
commas are for dweebs. fixed.